external/tools/tools.depproj

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
+
+  <PropertyGroup>
+    <EnableBinPlacing>false</EnableBinPlacing>
+    <NugetRuntimeIdentifier>$(ToolRuntimeRID)</NugetRuntimeIdentifier>
+    <RidSpecificAssets>true</RidSpecificAssets>
+    <NoWarn>$(NoWarn);NU1603;NU1605</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.ILAsm">
+      <Version>$(MicrosoftNetCoreIlasmPackageVersion)</Version>
+    </PackageReference>
+    <PackageReference Include="Microsoft.NETCore.ILDAsm">
+      <Version>$(MicrosoftNetCoreIlasmPackageVersion)</Version>
```

Comment:
Is there ever a case where these version numbers will be different?  I see we are using the ILAsm version number to get the ILDAsm package.

Reply:
They should always be the same.